codecov.yml

```diff
@@ -1,2 +1,5 @@
 fixes:
   - "pennylane_qchem/::qchem/pennylane_qchem/"
+
+ignore:
+  - "pennylane/plugins/tests/*"
```

Comment:
Hmm, I wonder how coverage is going to be affected by having the shared test suite. Mostly for plugins :thinking: Might be that we'll have to think of a "shared coverage" option?

Reply:
Once we have the CI executing the shared tests, the coverage will be fine 🙂 With tests, all you need to do is execute them and they'll be counted for full coverage (and `default.gaussian`, `default.qubit`, `default.qubit.tf` should be able to execute all tests)